SU2_CFD/include/gradients/computeGradientsLeastSquares.hpp

```diff
@@ -105,20 +107,12 @@ void computeGradientsLeastSquares(CSolver* solver,
       /*--- Distance vector from iPoint to jPoint ---*/
 
       su2double dist_ij[MAXNDIM] = {0.0};
-
-      for (size_t iDim = 0; iDim < nDim; ++iDim)
-        dist_ij[iDim] = coord_j[iDim] - coord_i[iDim];
+      GeometryToolbox::Distance(nDim, coord_j, coord_i, dist_ij);
```

Comment:
same suggestion to return the distance from the routine instead of an in/out arg

Reply:
Same reply as to why it can't be done.